ui/widgets/playercontrols.go

```diff
@@ -105,8 +110,9 @@ func NewPlayerControls() *PlayerControls {
 	pc.prev = widget.NewButtonWithIcon("", theme.MediaSkipPreviousIcon(), func() {})
 	pc.next = widget.NewButtonWithIcon("", theme.MediaSkipNextIcon(), func() {})
 	pc.playpause = widget.NewButtonWithIcon("", theme.MediaPlayIcon(), func() {})
+	pc.loop = widget.NewButtonWithIcon("", themedResReplay, func() {})
 
-	buttons := container.NewHBox(pc.prev, pc.playpause, pc.next)
+	buttons := container.NewHBox(pc.prev, pc.playpause, pc.next, pc.loop)
```

Comment:
What do you think about putting the loop button underneath the volume control slider? That was where I envisioned it would go, along with the eventual shuffle toggle and hopefully DLNA cast control. I like the symmetry of the play button being in the center, and I also might add skip forward/back buttons eventually to go +- some amount of seconds within the track

Reply:
I'm OK with any of these approaches. I was trying to reproduce the same interface the Spotify client has, with the shuffle button to the left and the loop button to the right (which would still maintain symmetry).

However, I can see that Navidrome has this button closer to the volume slider, so we can go with that approach instead. If you are planning to add more buttons to this container, then having shuffle and loop in here will be too much!